packages/nimble-components/src/table/models/virtualizer.ts

```diff
@@ -110,4 +113,31 @@ export class Virtualizer<TData extends TableRecord = TableRecord> {
 
         this.rowContainerYOffset = rowContainerYOffset;
     }
+
+    private notifyFocusedCellRecycling(): void {
+        let tableFocusedElement = this.table.shadowRoot!.activeElement;
+        if (tableFocusedElement) {
```

Comment:
Do you need this if-statement? It looks like the same condition will be checked in the while-loop, so it seems like it would be fine to execute the code inside the body of the if-statement regardless of the initial value of `tableFocusedElement`.

Reply:
You're right, the outer `if` was redundant since that condition was part of the `while` - removed the outer `if`.